editor/block-mover/style.scss

```diff
@@ -3,7 +3,6 @@
 	top: 10px;
 	left: 0;
 	padding: 0 14px 20px 0; // handles hover area
-	z-index: z-index( '.editor-block-mover' );
```

Comment:
Thanks for the response! I think the drag and drop ability for blocks within Gutenberg is definitely worth pursuing, and consensus in the community surrounding this, ultimately architectural, issue would be great, so the direction for accomplishing this could be seen as consistent and uniform. Just my two cents.